test/Microsoft.ML.Tests/CSharpCodeGen.cs

```diff
@@ -15,7 +15,7 @@ public CSharpCodeGen(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Fact(Skip = "Temporary solution(Windows ONLY) to regenerate codegenerated CSharpAPI.cs")]
+        [Fact]
```

Comment:
I believe you did not intend to keep this.

Reply:
Actually, no, I want to have test which validates CSharpApi

---
In reply to: [194211022](https://github.com/dotnet/machinelearning/pull/339#discussion_r194211022) [](ancestors = 194211022)